src/app/machines/views/MachineList/MachineList.tsx

```diff
@@ -37,6 +37,33 @@ const parseFilters = (filters: Filters): FetchFilters => {
   return fetchFilters;
 };
 
+const ErrorsNotification = ({
+  errors,
+  onAfterDismiss,
+}: {
+  errors: APIError;
+  onAfterDismiss?: () => void;
```

Comment:
If this doesn't get used by the other notifications (see my comment below) it looks like you could remove this prop as it isn't being used.